packages/icon-button/src/IconButton.tsx

```diff
@@ -129,10 +135,12 @@ const iconButtonVariants: { readonly [K in Variant]: string } = {
 const disabledStyle: { readonly [K in Variant]: string } = {
   [Variant.Light]: css`
     color: ${uiColors.gray.light2};
+    pointer-events: none;
```

Comment:
What about these changes?

Reply:
Handles disabling pointer events when we're no longer conditionally implementing the onClick